sympy/physics/mechanics/kane.py

```diff
@@ -40,13 +42,17 @@ class KanesMethod(_Methods):
         equations used to solve for non-contributing
         forces.
     mass_matrix : Matrix
-        The system's mass matrix
+        The system's mass matrix for the u's
     forcing : Matrix
-        The system's forcing vector
+        The system's forcing vector for the u's
     mass_matrix_full : Matrix
         The "mass matrix" for the u's and q's
     forcing_full : Matrix
         The "forcing vector" for the u's and q's
+    mass_matrix_kin : Matrix
+        The "mass matrix" for the q's
+    forcing_kin : Matrix
+        The "forcing vector" for the q's
```

Comment:
It'd be nice to come up with better names for this. They are kinematical differential coefficient and remainder matrices. I'm not sure what a nice concise, informative name would be though.

Reply:
it's fair that there is really no "mass" associated with these equations, but I kept the name to  be consistent with the `mass_matrix_full` (which has as subset of it these equations)

Happy to use something else, but I'll defer to you for a better name